config/schedule.rb

```diff
@@ -25,15 +25,15 @@
 # setup
 daily_schedule = ENV['DAILY_SCHEDULE'] || '3:10 am'
 import_schedule = ENV['IMPORT_SCHEDULE'] || '5:30 pm'
-glacier_import_schedule = ENV['IMPORT_SCHEDULE'] || '5:30 pm'
+# glacier_import_schedule = ENV['IMPORT_SCHEDULE'] || '5:30 pm'
```

Comment:
Whya re these removed?

Reply:
The glacier backups can actually be removed, we are using a different mechanism now, but I agree, seems like the wrong PR for that.

Reply:
The rubocop CI check complains otherwise since these variables are not used. Should I revert the change and just ignore the CI check for now?